docs/tempo/website/getting-started/_index.md

```diff
@@ -1,60 +1,60 @@
 * [Zipkin](https://zipkin.io/pages/tracers_instrumentation)
 * [OpenTelemetry](https://opentelemetry.io/docs/concepts/instrumenting/)
 
-#### OpenTelemetry Auto Instrumentation
+### OpenTelemetry auto-instrumentation
 
 Some languages have support for auto-instrumentation. These libraries capture telemetry
 information from a client application with minimal manual instrumentation of the codebase.
 
-* [OpenTelemetry Java Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation)
-* [OpenTelemetry .NET Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)
-* [OpenTelemetry Python Autoinstrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib)
+* [OpenTelemetry Java auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation)
+* [OpenTelemetry .NET auto-instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)
+* [OpenTelemetry Python auto-instrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib)
 
-> Note: Check out our [instrumentation references]({{< relref "./instrumentation" >}}) to learn how to instrument your
-> favourite language for distributed tracing.
+> **Note**: Check out the [instrumentation references]({{< relref "./instrumentation" >}}) to learn how to instrument your
+> favorite language for distributed tracing.
 
-## 2. Pipeline (Grafana Agent)
+## Pipeline (Grafana Agent)
 
-Once your application is instrumented for tracing, the next step is to send these traces
-to a backend for storage and visualization. It is common to build a tracing pipeline that
-offloads spans from your application, buffers them and eventually forwards them to a backend. Tracing
-pipelines are optional (most clients can send directly to Tempo), but you will find that
-they become more critical the larger and more robust your tracing system is.
+Once your application is instrumented for tracing, the traces need to be sent
+to a backend for storage and visualization. You can build a tracing pipeline that
+offloads spans from your application, buffers them, and eventually forwards them to a backend.
+Tracing pipelines are optional (most clients can send directly to Tempo), but the pipelines
+become more critical the larger and more robust your tracing system is.
 
 The Grafana Agent is a service that is deployed close to the application, either on the same node or
-within the same cluster (in kubernetes) to quickly offload traces from the application and forward them to
-a storage backend. It also abstracts features like trace batching and backend routing away from the client.
+within the same cluster (in Kubernetes) to quickly offload traces from the application and forward them to
+a storage backend.
+The Grafana Agent also abstracts features like trace batching and backend routing away from the client.
```

Comment:
Does it also add retry logic?

Reply:
@hedss  do you know if it does?

Reply:
Yes, this uses the full OTel retry on failure configuration options. So maybe we should say:
```
The Grafana Agent also abstracts features like trace batching to a remote trace backend store, including retries on write failures.
```

Reply:
Thank you. I've updated the text to reflect your suggestion.